### ISSUE TITLE
DOC: Negative binomial formulation confusion

### ISSUE THREAD

**Issue by dezmondgoff** [numpy/numpy#10454]
Currently, there are two formulations in the documentation: (1) in the `Returns` description, some form of "N trials given n successes" where the last trial is incorrectly indexed as N + n and (2) in the `Notes`, "N failures given n successes and p(success) = p". ~~However, neither of these descriptions match the variable that is actually returned, which via the gamma-Poisson mixture is "N successes given n failures and p(success) = p".~~ While "N failures given n successes" and "N successes given n failures" have the same expression as their respective pmfs, they are not equivalent except when p=0.5:

```
import numpy as np
import scipy as sp
import scipy.special

N = np.arange(100)
p = 0.5
# N successes given 5 failures
p0 = sp.special.binom(N + 5 - 1, N) * p**N *(1-p)**5
# N failures given 5 succeses
p1 = sp.special.binom(5 + N - 1, 4)  * p**5 *(1-p)**N
np.allclose(p0 ,p1)
# True

p = 0.75
p0 = sp.special.binom(N + 5 - 1, N) * p**N *(1-p)**5
p1 = sp.special.binom(5 + N - 1, 4)  * p**5 *(1-p)**N
np.allclose(p0, p1)
# False
```

Clearly "N failures given n successes and p(success) = p" is equivalent to "N successes given n failures and p(success) = 1 - p". ~~The quickest solution is to change the description to match  "N successes given n failures and p(success) = p".~~ I also suggest switching to the p(k; r, p) notation used on the negative binomial distribution Wikipedia page. 

EDIT: On further inspection, the output does match the second description, given the shape and scale parameterization of the gamma distribution. I've changed the title to reflect this.

**Comment by mattip** [comment]
Cool. Please make sure you work according to our [dev guidlines](http://www.numpy.org/devdocs/dev/index.html). Isn't this a duplicate of #10756, which was closed via pull request #10883? You can view the rendered documentation after that PR [here](http://www.numpy.org/devdocs/reference/generated/numpy.random.negative_binomial.html)

**Comment by manasbedmutha98** [comment]
@mattip, I didn't understand if there's anything left to be done after #10883. Can you please help out here?

**Comment by mattip** [comment]
Could you take a look at the [new documentation](http://www.numpy.org/devdocs/reference/generated/numpy.random.negative_binomial.html)? If it reads well, then you can close the issue, thanks

**Comment by dezmondgoff** [comment]
I think the new documentation looks correct.
